less/home-page.less

```diff
@@ -72,4 +70,93 @@
       }
     }
   }
-}// home
\ No newline at end of file
+}
+
+button{
+  .buttonTag
+
+}
+&:hover{
+    background-color: white;
+    color: black;
+  }// home
+
+body{
+
+  @media @phone{ 
+
+.space-images {
+    display:flex;
+    justify-content: center;
+    
+    .blast{
+      margin: -.1%;
+      width: 95%;
+      padding: 2%;
+      border-radius: 20px;
+    }
+    }
+    .sky{
+      display:none;
+    }
+    .launch{
+      display:none;
+    }
+.walk{
+  display: block;
+}
+    .walk-img{
+    // display:column;
+    // justify-content: center;
+    margin: -.1%;
+      width: 95%;
+      padding: 2%;
+      border-radius: 20px;
+
+    }
+
+h2{
+  display: left;
+  margin-left:-350px;
+}
+		// flex-wrap: wrap;
+		// box-sizing: border-box;
+		// align-items: stretch;
+
+    // flex-direction: column;
+    // align-items: center;
+
+}
+}
+
+.container home {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0%;
+  height: auto;
+}
+
+@media @phone{
+      display:flex;
+      flex-direction: column-reverse;
+      padding: 5px;
+      align-items: center;
+      margin-top: 30px;
+      height: auto;
+
+      .walk-text{
+        width: 50%;
```

Comment:
This should not be in the `phone` break point. This is forcing your `walk-text` to be half the size of the parent section `walk` which is the reason the text is squeezed in the bottom section. At the mobile break point this as well as the div containing the image should be set to 99% if not 100 to match the design files.